src/Components/ServiceScene/Breakdowns/ByFrameRepeater.tsx

```diff
@@ -43,11 +43,25 @@ export class ByFrameRepeater extends SceneObjectBase<ByFrameRepeaterState> {
     });
   }
 
+  private getSortedSeries(data: PanelData) {
```

Comment:
Can we add a comment that this is working under the assumption that each dataframe has a single time and value field

Reply:
Each data frame from this `PanelData` is a series, I'm not sure I understand the comment. In any case, I can add this to the follow up changes.